### ISSUE TITLE
SPID/KID support needed for merges

### ISSUE THREAD

**Issue by alandtse** [matortheeternal/zedit#256]
Many mods are using [SPID](https://www.nexusmods.com/skyrimspecialedition/mods/36869) and [KID](https://www.nexusmods.com/skyrimspecialedition/mods/55728) now to distribute items by formID. 

Does this mean that zedit should start processing .ini files to also identify changed formids because of merges? If not doing it directly in zedit, is there an output we can get from zedit so we can script a regex replace on those ini files?

**Comment by BrentW89** [comment]
Should I remove SPID mods from my zMerge? I just assumed they would work, but you're saying I'm going to have problems?

**Comment by alandtse** [comment]
Yes, SPID requires the ID and the mod name not to change so don't merge if you want it to work.

For those in VR, I've released a new SPID VR version that handles this (Nexus release will happen when uploads work again).
https://github.com/alandtse/Spell-Perk-Item-Distributor/releases/tag/5.2.0.1
